### ISSUE TITLE
Documentation does not reflect latest version.

### ISSUE THREAD

**Issue by VictorEijkhout** [BlueBrain/HighFive#923]
On page https://bluebrain.github.io/HighFive/
there is a statement
```
Note: H5File.hpp is the top-level header of HighFive core which should be always included.
```
which has no bearing on the actual example. That's a 2.7 / 2.8 mismatch.

**Comment by 1uc** [comment]
Thank you for letting us know.

Correct would be as of 2.8, please consider using `highfive.hpp` since we'll promise to include everything via that header. Until 3.0 we'll do our best to not reduce what `H5File.hpp` includes. As of 3.0 we'd like the freedom to make `H5File.hpp` only promise that it includes `HighFive::File`.

Hence, if compatibility with 2.8 and newer, including 3.0, is desired `<highfive/highfive.hpp>` is a short sensible choice. If compatibility with `<3`, including `2.7` and older is required use `H5File.hpp`. If compatibility across all version is important. The best bet is to include each associated header in `highfive/H5{File,Attribute,Dataset,...}.hpp`, but nothing in `highfive/bits` or ending in `_misc.hpp`.

**Comment by alkino** [comment]
Fixed by #931
